kroxylicious-api/src/main/java/io/kroxylicious/proxy/filter/FilterFactoryContext.java

```diff
@@ -19,13 +19,20 @@
 public interface FilterFactoryContext {
 
     /**
-     * The event loop that this Filter's channel is assigned to.
+     * @deprecated replaced with {@link #filterThreadExecutor()}
+     * @see #filterThreadExecutor()
+     */
+    @Deprecated(since = "0.5.0", forRemoval = true)
+    ScheduledExecutorService eventLoop();
+
+    /**
+     * An executor service backed by the filter dispatch thread used
+     * to create Filter instances and invoke their methods. It should be
+     * safe to mutate Filter member state from this executor.
```

Comment:
The first sentence is a bit hard to read. How about?
```suggestion
     * An executor service which delegates to the same thread that is used to dispatch the filter methods.
       It is safe to mutate Filter member state from code run by this executor.
```